Bug Scrub Announce blog post (#360)
fluxcd/website: content/en/blog/2021-06-28-bug-scrub-announce.md

---
author: kingdonb
date: 2021-06-28 08:30:00+00:00
title: Flux Bug Scrub Announce
description: We are starting the Flux Bug Scrub initiative. A great way to help out in the community and get started by learning more about Flux and interacting with user issues!
url: /blog/2021/06/flux-bug-scrub-announce/
---

## What is the Flux Bug Scrub

In the past months we have had many folks reach out to us and ask where
they could get involved, make their first steps in the Flux community
and give back in a way.

We started looking around and stole a concept from SIG Node (who also

> stole it from SIG CLI) over in the Kubernetes community. Because for us

Reviewer: Let's use friendlier words not `stole`..

Author: :+1:

> Because for us a great way to get started is to learn more about Flux

Reviewer: ```suggestion
For us, a great way to get started, is to learn more about Flux
```

> through direct experience, when e.g. trying to reproduce issues from

Reviewer: ```suggestion
through direct experience, when e.g. trying to reproduce issues reported by
```

other Flux users, and the general business of chopping wood and
carrying water. The **Flux Bug Scrub** is born.

## Here is how it works

Show up at the appointed time, if you can. The first Bug Scrub is

- June 30 12:00 UTC, 8:00AM ET.
- Meet at \#flux on the [CNCF Slack](https://slack.cncf.io),

we will provide a Zoom link closer to the event time.

> We will have one or more contributors with Triage access in attendance,

Reviewer: ```suggestion
We will have one or more contributors with triage access in attendance,
```

> and attempt to visit each issue in the target group. (The targets will

Reviewer: ```suggestion
and attempt to review each issue in the target group. (The targets will
```

vary by week, or by event date, as this will be a recurring event on
some TBD schedule.) A key goal is to figure out if a given issue can be
reproduced, and report our findings. Some issues don\'t provide enough
info to get us all the way to reproduction, so we may ask more questions
and wait for the submitter to respond.

If someone present wants to volunteer to follow up an issue, we will
make an assignment then. If all we can do is read the thread once and
confirm that labels are still correct, (or fix them up when they were

> not,) then it will be providing a valuable service to Flux maintainers

Reviewer: ```suggestion
not) then it will be providing a valuable service to Flux maintainers
```

and the community at large. A process will be demonstrated to avoid
backtracking from week to week and maximize coverage. This is not a race
to close the issues, but if an issue has received no reply and it looks

> like the submitter has lost interest, it can be closed.

Reviewer: ```suggestion
like the submitter has lost interest, it could be closed.
```
